nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -1717,6 +1717,8 @@ private boolean mayBeNullExpr(VisitorState state, ExpressionTree expr) {
         exprMayBeNull = mayBeNullFieldAccess(state, expr, exprSymbol);
         break;
       case IDENTIFIER:
+        // ToDo: Also run handler.onOverrideMayBeNullExpr BEFORE dataflow, rather than after, in
+        // these two cases?
```

Comment:
Not clear what to do here until we have handlers that actually override the nullability of fields.